src/components/notifications/WebhookConfigModal.tsx

```diff
@@ -0,0 +1,386 @@
+                {keys.map((atrribute, index) => (
+                    <div className="w-100-imp cn-7 fs-12 mb-8 flex left hover-trigger" data-testid={`${this.state.webhookAttribute[atrribute]}-${index}`}>
+                        <span className="bcn-1 br-4 fs-12 fw-4 lh-16 p-4">{this.state.webhookAttribute[atrribute]}</span>
+                        <Tippy
+                            className="default-tt"
+                            arrow={false}
+                            placement="bottom"
+                            content={this.state.copyAttribute ? 'Copied!' : 'Copy'}
+                            trigger="mouseenter click"
+                            onShow={(_tippy) => {
+                                setTimeout(() => {
+                                    _tippy.hide()
+                                    this.setCopied(false)
+                                }, 4000)
+                            }}
+                            interactive={true}
+                        >
+                            <Clipboard
+                                data-value={this.state.webhookAttribute[atrribute]}
+                                className="ml-8 pointer hover-only icon-dim-16"
+                                onClick={this.copyToClipboard}
+                            />
+                        </Tippy>
+                    </div>
+
+                ))}
+            </div>
+        )
+    }
+
+    renderHeadersList() {
+        return (
+            <div>
+                <div className="mb-8">
+                    {this.state.form.header?.map((headerData, index) => (
+                        <CreateHeaderDetails
+                            key={`tag-${index}`}
+                            index={index}
+                            headerData={headerData}
+                            setHeaderData={this.setHeaderData}
+                            removeHeader={this.removeHeader}
+                            headerIndex={3 + (index + 2)}
+                        />
+                    ))}
+                </div>
+            </div>
+        )
+    }
+
+    renderWithBackdrop(body) {
+        return <Drawer position="right">
+            <div className="h-100 modal__body w-885 modal__body--p-0 dc__no-top-radius mt-0 dc__position-rel">
+                <div className="h-48 flex flex-align-center dc__border-bottom flex-justify bcn-0 pb-12 pt-12 pl-20 pr-20">
+                    <h1 className="fs-16 fw-6 lh-1-43 m-0 title-padding">Configure Webhook</h1>
+                    <button type="button" className="dc__transparent" onClick={this.props.closeWebhookConfigModal}>
+                        <Close className="icon-dim-24" />
+                    </button>
+                </div>
+                {body}
+            </div>
+        </Drawer>
+    }
+
+    render() {
+        let body;
+        if (this.state.view === ViewType.LOADING) {
```

Comment:
create function which return the jsx inside if else conditions

Reply:
done